Python/facility_data_demo.py

```diff
@@ -1,35 +1,52 @@
 import requests
+import sys
+from datetime import datetime
 import pandas as pd
+import warnings
 
-# These are examples of querying facility data.
+#######
+# NOTE: If you are looking to download data in bulk, please see the bulk_data_demo.py file.
+# An example of a bulk requests is downloading all facility data for more than one year.
+#######
 
-####### Streaming services API #######
-
-# If you have a lot of data to request, use the streaming services API to avoid
-# pagination and much quicker collection of the data you need.
+# Below are examples of querying facility attribute data.
 
+# Set your API key here
 API_KEY = 'YOUR_API_KEY'
 
+####### Streaming services API #######
+# This is an example of how to use the streaming services API to get facility attribute data.
+
+# Use this API for continuous data streams and avoiding paging through results.
+
 # api parameters for the streaming facility/attributes endpoint
 parameters = {
     'api_key': API_KEY,
-    'year': 2020,
+    'year': '2020|2021',
     'unitFuelType': 'Coal|Natural Gas'
 }
 
+if len(parameters['year']) > 4:
```

Comment:
This isn't parsing the 2020|2021 as a list, but instead counting the length of the string. You may want to check this another way.